src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm \n" + logo );
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        Scanner scanner = new Scanner(System.in);
+        String text = "";
+        Boolean isContinue = true;
+
+        Todo[] todos = new Todo[100];
```

Comment:
In this program, you use Todo as the parent class for Deadline and Event. I found this a bit misleading as the todo, event, and deadline should be three tasks of the same level (all children of the parent class Task). This is just a suggestion.